hax/hax/handler.py

```diff
@@ -167,17 +167,6 @@ def _do_work(self, planner: WorkPlanner, motr: Motr):
 
                     LOG.debug('Got %s message from planner', item)
                     if isinstance(item, FirstEntrypointRequest):
-                        LOG.debug('first entrypoint request, broadcast FAILED')
```

Comment:
Why did we do it in the 1st place, and what are we doing now instead?

Reply:
We were sending it because it was the safest place to report a process failure as hax received an entrypoint request. Consul notifications can be delayed and process can already be starting (entrypoint was already replied to). Hax cannot report a starting process as failed as that leads to other in-consistencies.
I still think reporting FAILED on receiving a first entrypoint request is the right place but due to out-of-order starting and with multiple ioservices on a node, Failed of one process may reach other process during its startup. This leads to the PV dirty issue where if no clean pool versions are found during dix setup motr process crashes.
Problem needs to be solved but we need some additional work there.